protocol/0014-ORDT-order_types.md

```diff
@@ -90,6 +142,50 @@ Network orders are used during [position resolution](./0012-POSR-position_resolu
   - A GTT order submitted at a time >= its expiry time is rejected. (<a name="0014-ORDT-005" href="#0014-ORDT-005">0014-ORDT-005</a>)
 - No party can submit a [network order type](#network-orders)  (<a name="0014-ORDT-006" href="#0014-ORDT-006">0014-ORDT-006</a>)
 
+### Stop orders
+
+- A stop order containing both a trigger price and a trailing stop distance will be rejected.(<a name="0014-ORDT-007" href="#0014-ORDT-007">0014-ORDT-007</a>)
```

Comment:
This is represented by a oneof through the API, so not possible / testable because of the constraint of the API, should this be removed?